### ISSUE TITLE
`_NET_WM_WINDOW_TYPE_DIALOG` missing from window parameters, i3-wm does not correctly float the popup dialog.

### ISSUE THREAD

**Issue by kevenwyld** [ProtonMail/proton-bridge#297]
Issue tracker is ONLY used for reporting bugs with technical details. "It doesn't work" or new features should be discussed with our customer support. Please use bug report function in Bridge or contact bridge@protonmail.ch.

## Expected Behavior
When left clicking on the bridge tray icon a dialog should open presenting the window manager with `_NET_WM_WINDOW_TYPE_DIALOG` allowing tiling window managers to correctly apply automatic floating rules to it.  (eg. i3-wm)

## Current Behavior
i3 opens a window and attempts to tile it, sometimes filling the whole screen and making the buttons hard to locate and read.  See screenshot further down.

## Possible Solution
I am not too familiar with the protonmail-bridge codebase or go in general, and from poking around in the dialog window code it appears to be classified correctly, however it is missing the necessary hints to allow tiling window managers to handle it appropriately.

Other similar windows, Nextcloud client in this case was the closest to test with, have slightly different X11 properties and float correctly (xprop used to view these parameters):

nextcloud:
```
_NET_WM_STATE(ATOM) = _NET_WM_STATE_FOCUSED
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_DIALOG, _KDE_NET_WM_WINDOW_TYPE_OVERRIDE, _NET_WM_WINDOW_TYPE_NORMAL
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
```

bridge:
```
_NET_WM_STATE(ATOM) = _NET_WM_STATE_ABOVE, _NET_WM_STATE_STAYS_ON_TOP
_NET_WM_WINDOW_TYPE(ATOM) = _KDE_NET_WM_WINDOW_TYPE_OVERRIDE, _NET_WM_WINDOW_TYPE_NORMAL
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
```

## Steps to Reproduce
1.  Start i3-session
2.  Start protonmail-bridge
3.  Left click on protonmail-bridge 
4.  Window will fill whatever space is available in the current tiling scheme.

## Version Information
2.3.0

## Context (Environment)
OS: Archlinux
Window manager: i3-wm

## Detailed Description
Here's a screenshot of what the window looks like. It expands beyond the limit of it's GUI elements and looks very strange.  Other dialogs from other applications that are similar present themselves as dialogs correctly using X11 window hints so i3 does not try to tile them.  I only have this problem with protonmail-bridge so I believe this is the appropriate place to open an issue.

![Screenshot_2022-09-25_21-29-14_obfuscated](https://user-images.githubusercontent.com/11936535/192189125-f2ba4704-1efb-49c6-84e3-cbb03df2cffe.jpg)

It's entirely possible that my lack of experience with creating GUI applications has made me completely misunderstand this issue and provide all sorts of irrelevant information. This is still a problem on all the machines with i3-wm that I have used bridge on so far.  And every other tray stile application does work, so I'm pretty sure there is a problem here.

Thanks!

**Comment by andrzejsza** [comment]
yes, there is a problem. Bridge should request a floating window, but only for i3. we have internal ticket for this (GODT-1427) but can't promise any speedy resolution.

**Comment by kevenwyld** [comment]
@andrzejsza thanks!  I appreciate the response and totally understand it's not a high priority.  I'm glad to hear you are tracking it though.

**Comment by deescuderoo** [comment]
@kevenwyld Your solution is a great idea, thanks for sharing. It's a shame it still captures the actual protonmail-bridge app when you open it, but that's a minor side effect. This is a good workaround for now.

**Comment by ni-ka** [comment]
@andrzejsza I am having the same issue with the gnome extension from pop OS - prob every popOS user will have this issue, I use manjaro however.
![Screenshot from 2023-01-11 17-16-45](https://user-images.githubusercontent.com/1225776/211876002-06395f82-d5c9-4c1c-b29c-838ef061531a.png)

Even after disabling the extension the window stayed the same, and I had to restart the bridge with the extension disabled for it to be displayed normally.

Based on @kevenwyld 's workaround (thanks!) I was able to add a floating window exception. 
![Screenshot from 2023-01-11 17-21-59](https://user-images.githubusercontent.com/1225776/211876955-db990e1f-108f-4bf8-a2c7-660499211763.png)

Might be good to include a section in the README if there is no plans to fix this for now.

**Comment by LBeernaertProton** [comment]
@kevenwyld  @ni-ka did version 3.x address these issues?
